### PR TITLE
Plant Disease Detection Rover

### PR DESCRIPTION
**Pull request by Akshay-Vs**
## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
@Akshay-Vs thank you so much for sharing your project idea! i'm wondering, 
- do you have a broad idea for what kind of code you'll need to write?
- when you say design the circuit for the rover, do you mean connecting the motors, the camera, etc. to the pi? if so, is there an existing circuit you plan on modelling after?
- is there a place where you'll be able to use your rover and test it?

also, could you please recommit the exu3.md file? thank you sm!!

**Comment by Akshay-Vs** [comment]
- Yes, To develop this rover using OpenCV, I'll need to collect and preprocess images of healthy and diseased plants, split the dataset into training and test sets, train a machine learning model such as a CNN on the training set, evaluate the model's performance on the test set, and use the trained model to classify new images of plants as healthy or diseased.

- Currently, I don't yet have a circuit diagram, but I have some ideas about how to connect the hardware components together. I'll need to design a circuit diagram after writing the software.

- The rover is about 18 x 24 inch size, so finding a place to test is quite easy

- Somehow, I cannot recommit the md file, if you need I'll open a new pull request, Thanks for considering!

**Comment by bellesea** [comment]
> * Yes, To develop this rover using OpenCV, I'll need to collect and preprocess images of healthy and diseased plants, split the dataset into training and test sets, train a machine learning model such as a CNN on the training set, evaluate the model's performance on the test set, and use the trained model to classify new images of plants as healthy or diseased.
> * Currently, I don't yet have a circuit diagram, but I have some ideas about how to connect the hardware components together. I'll need to design a circuit diagram after writing the software.
> * The rover is about 18 x 24 inch size, so finding a place to test is quite easy
> * Somehow, I cannot recommit the md file, if you need I'll open a new pull request, Thanks for considering!

hiii @Akshay-Vs thanks so much for your rapid response! if possible, could you please add the details to above to your proposal? you can do so by clicking on the pen icon on your page [here](https://github.com/Akshay-Vs/winter/blob/main/akshay.md) then clicking commit change after :)

on recommitting the file, if you click the merge button on this [pull request i made](https://github.com/Akshay-Vs/winter/pull/1) it would work :)

please let me know if you have any questions! thank you :)

**Comment by Akshay-Vs** [comment]
[Here](https://github.com/hackclub/winter/pull/74) is the updated pull request. I tried to make some updates to this pull request, but the edit button is disabled. Is there anything I can do to resolve this issue?

**Comment by bellesea** [comment]
> [Here](https://github.com/hackclub/winter/pull/74) is the updated pull request. I tried to make some updates to this pull request, but the edit button is disabled. Is there anything I can do to resolve this issue?

thank you so much for making another PR! closing this one and will review the new one, have an awesome day :)
